src/Standards/BestIt/Sniffs/Functions/ForbiddenFunctionsSniff.php

```diff
@@ -31,22 +31,24 @@ class ForbiddenFunctionsSniff extends BaseSniff
     /**
      * If true, an error will be thrown; otherwise a warning.
      *
+     * @phpcsSuppress SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingNativeTypeHint
      * @var boolean
      */
     public $error = false;
 
     /**
      * The used file.
      *
-     * @var File|void
+     * @var File|null
      */
-    protected $file;
+    protected File|null $file;
```

Comment:
Missing null initialization here. Seems to be the last one. :-)

Reply:
good eye.